src/backend/utilities/batchSignAndSubmitAttestation.ts

```diff
@@ -106,6 +113,9 @@ async function createPendingTransaction() {
     Blockchain.signAndSubmitTx(authorized, identity),
   );
   logger.debug('Transaction submitted');
+
+  // Not sure if this is the right place
+  removeFromBlackList(submittableRevocations);
```

Comment:
There’s error handling in this file as well, I think this belongs there with the added checks that these transactions were actually successfully submitted.

Reply:
Do you mean that `try-catch`?